docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -185,6 +185,17 @@ spec:
 ---
 apiVersion: v1
 kind: query
+spec:
+  name: Get Ubuntu Advantage (Ubuntu Pro) Attachment Status
```

```suggestion
  name: Get Ubuntu Advantage (Ubuntu Pro) attachment status
```

"attachment status" isn't a proper noun right? If not, we want to use sentence case: https://fleetdm.com/handbook/company/why-this-way#why-does-fleet-use-sentence-case